distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -226,6 +226,20 @@ type Duplicate_Type_Selector
     to_display_text : Text
     to_display_text self = "Duplicate type selector for column " + self.column + "."
 
+## Indicates that the column was already present in the table.
+type Existing_Column
+    Error column_name
+
+    to_display_text : Text
+    to_display_text self = "The column '" + self.column_name + "' already exists."
+
+## Indicates that the column was not present in the table.
+type Missing_Column
+    Error column_name
+
+    to_display_text : Text
+    to_display_text self = "The column '" + self.column_name + "' was not found."
```

Comment:
I don't want to be too picky, but thinking about clarity of errors, I've got the following thoughts:

These errors sound nice as they can be pretty generic, we can re-use them - _nice!_

But then, I thought that actually this is not best - maybe we should be creating more specialized error types for specific circumstances - then the error message can also be more precise. IMO this could be a good direction (as a general rule of thumb in the future). What do you think?

Here, for example, I'd consider adding to the messages of these errors something like:
```
"The column '" + self.column_name + "' already exists, but `Set_Mode.Update` was selected."
```

and analogously for the other one. Then it will be clear where the error originated from and how this one differs, for example, `Missing_Input_Columns` - which on the first look seems like it is basically the same as `Missing_Column`. I'm in no way suggesting to merge them though - on the contrary - I think we should make them more distinct by making them more specific to the specific error condition that they are indicating.

(This is also very much a message to myself: I think we should try as much as possible to keep the error messages as helpful as possible to the user, so that reading it it is clear what went wrong and what can be done to fix it; in the example above - the user is told that a specific `Set_Mode` was selected, implying that they can try to choose a different mode (or rename, depending on what they need)).